app/templates/discover.hbs

```diff
@@ -70,6 +70,19 @@
             </select>
         </div>
     </div>
+    <div class="row visible-xs">
```

Comment:
Do you think listing the filters at the top could be useful on larger screens, too? Or is that too redundant?